runtime/queries/metricsview_aggregation.go

```diff
@@ -329,6 +332,9 @@ func (q *MetricsViewAggregation) rewriteToMetricsViewQuery(export bool) (*metric
 		res.IsoDuration = q.ComparisonTimeRange.IsoDuration
 		res.IsoOffset = q.ComparisonTimeRange.IsoOffset
 		res.RoundToGrain = metricsview.TimeGrainFromProto(q.ComparisonTimeRange.RoundToGrain)
+		if q.ComparisonTimeRange.TimeZone != "" && qry.TimeZone == "qry.TimeZone" {
```

Comment:
Why would it be `"qry.TimeZone"` as a string? It's expected to be an IANA time zone code.

Also since we don't support different time zones in `qry.TimeZone`, IMO it should return an error if `q.TimeRange.TimeZone` and `q.ComparisonTimeRange.TimeZone` specify different time zones.